engine-types/src/parameters.rs

```diff
@@ -53,14 +54,14 @@ pub struct PromiseBatchAction {
 /// withdraw NEAR eth-connector call args
 #[derive(Debug, BorshSerialize, BorshDeserialize)]
 pub struct WithdrawCallArgs {
-    pub recipient_address: EthAddress,
+    pub recipient_address: Address,
     pub amount: Balance,
 }
 
 /// withdraw NEAR eth-connector call args
 #[derive(BorshSerialize, BorshDeserialize)]
 pub struct RefundCallArgs {
-    pub recipient_address: EthAddress,
-    pub erc20_address: Option<EthAddress>,
+    pub recipient_address: Address,
+    pub erc20_address: Option<Address>,
     pub amount: RawU256,
 }
```

Comment:
Could these break API? Looks like a manual Serialize and Deserialize is needed.

Reply:
No breaking changes. For the new `Address` type was implemented custom `Borsh Ser/Deserialize`. And it was tested. `H160`, `U256` do not support `Borsh`. Previously it was `type Adress = H160`.
So we need `RawAddress, RawU256`. But now we can remove at all `RawAddress`.
The code is much cleaner and logically true.

After that it will be PR for Borsh traits wrapped types for `U256`.